app/mailers/participant_validation_mailer.rb

```diff
@@ -105,6 +108,21 @@ def induction_coordinator_email(recipient:, school_name:, start_url:)
     )
   end
 
+  def induction_coordinator_check_ect_and_mentor_email(recipient:)
+    template_mail(
+      INDUCTION_COORDINATOR_CHECK_ECT_AND_MENTOR_TEMPLATE,
+      to: recipient,
+      rails_mailer: mailer_name,
+      rails_mail_template: action_name,
+      personalisation: {
+        statutory_guidance: STATUTORY_GUIDANCE_LINK,
+        sign_in: new_user_session_url,
+        step_by_step: step_by_step_url,
+        resend_email: resend_email_request_nomination_invite_url,
```

Comment:
Missing UTM params here. We'd normally pass these in as params so we can get the right campaign

Reply:
Added now